.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.1.0-beta.6
+current_version = 2.1.0-beta.7
```

Comment:
We should leave the bump version to a separate PR.

Reply:
Absolutely - This a rebase artifact.

Reply:
:heavy_check_mark: